html/schematics.py

```diff
@@ -237,7 +237,8 @@ def main():
 								tmpName = ingRow[4]
 								tmpLink = '<a href="' + ghShared.BASE_SCRIPT_URL + 'resourceType.py/' + ingRow[2] + '">' + tmpName + '</a>'
 							else:
-								tmpLink = '<a href="' + ghShared.BASE_SCRIPT_URL + 'resourceType.py/' + ingRow[2] + '">' + tmpName + '</a>'
+								tmpName = "Unknown Ingredient"
+								tmpLink = f"[{tmpName}]"
```

Comment:
I think this _might_ be a fix for #73 . See the comment here: https://github.com/pwillworth/galaxyharvester/issues/73#issuecomment-2684244787

Reply:
Nevermind, I checked and the link is still broken.